src/test/cli_test.ts

```diff
@@ -40,8 +41,21 @@ suite('cli', () => {
       return '';
     });
 
+    const closeServer = (server: http.Server) =>
+        new Promise((resolve, reject) => server.close((e: Error) => {
+          e ? reject(e) : resolve();
+        }));
+
     try {
-      await cliRun();
+      const serverInfos = await cliRun();
+      if (serverInfos) {
```

Comment:
Now that there's a test that tries to successfully launch some servers, we need to shut them down and wait or else we get port-in-use errors.